be/src/vec/exprs/vectorized_fn_call.cpp

```diff
@@ -104,12 +104,38 @@ doris::Status VectorizedFnCall::execute(VExprContext* context, doris::vectorized
     size_t num_columns_without_result = block->columns();
     // prepare a column to save result
     block->insert({nullptr, _data_type, _expr_name});
+    if (_function->can_fast_execute()) {
+        bool ok = fast_execute(context->fn_context(_fn_context_index), *block, arguments,
+                               num_columns_without_result, block->rows());
+        if (ok) {
+            *result_column_id = num_columns_without_result;
+            return Status::OK();
+        }
+    }
+
     RETURN_IF_ERROR(_function->execute(context->fn_context(_fn_context_index), *block, arguments,
                                        num_columns_without_result, block->rows(), false));
     *result_column_id = num_columns_without_result;
     return Status::OK();
 }
 
+bool VectorizedFnCall::fast_execute(FunctionContext* context, Block& block,
+                                    const ColumnNumbers& arguments, size_t result,
+                                    size_t input_rows_count) {
+    auto column_with_name = block.get_by_position(arguments[1]);
+    auto query_value = column_with_name.to_string(0);
+    std::string column_name = block.get_by_position(arguments[0]).name;
+    auto result_column_name = column_name + "_" + _function->get_name() + "_" + query_value;
```

Comment:
Save the unique identifier of expr in `ColumnValueRange` -> `TCodition filter` -> `ColumnPredicate`, so that `fast_execute` can find the result column of `_execute_all_compound_predicates`, is it possible?

Just an implementation idea, probably doesn't need modification

Reply:
In `SegmentIterator`, a lot of codes are used in order to splicing the sign string, and it seems that it is easier to understand with a unique identifier of expr

Reply:
It's a good idea. I will try it later